src/status_im/navigation/effects.cljs

```diff
@@ -190,7 +190,9 @@
  (fn [{:keys [content on-success on-error]}]
    (cond-> (share/open content)
      (fn? on-success) (.then on-success)
-     (fn? on-error)   (.catch on-error))))
+     :always          (.catch (fn [error]
+                                (log/error "[react-native-share]" error)
```

Comment:
A more verbose error log could be more useful e.g.
```clj
(log/error "Failed to share content with react-native-share" {:error error :effect :effects.share/open})
```

Reply:
Doesn't the error message depend on whether `on-success` throws as well? For example, we could be catching an unrelated exception because on-success called something incorrectly.

Reply:
yes you're right. Best remove references to react-native-share